### PR TITLE
Change xml:ids for inclusion in the admin guide (TEST DON'T MERGE)

### PR DESCRIPTION
**Pull request by cwickert**
### PR creator: Description

This is just a test for the sake of training

### PR creator: Are there any relevant issues/feature requests?

* bsc#...
* jsc#SLE-...

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/tuning_multikernel.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
    <xref linkend="sec-grub2-yast2-config"/> for more information.
   </para>
  </tip>
- <sect1 xml:id="sec-tuning-multikernel-enable">
+ <sect1 xml:id="sec-admin-multikernel-enable">
   <title>Enabling and configuring multiversion support</title>
```
Comment (cwickert) [inline xml/tuning_multikernel.xml:54]:
```suggestion
  <title>Enabe and configure multiversion support</title>
```

File: xml/tuning_multikernel.xml [hunk 2]
```diff
@@ -105,7 +105,7 @@
    </para>
   </warning>
 
-  <sect2 xml:id="sec-tuning-multikernel-enable-keep">
+  <sect2 xml:id="sec-admin-multikernel-enable-keep">
    <title>Automatically deleting unused kernels</title>
```
Comment (cwickert) [inline xml/tuning_multikernel.xml:109]:
```suggestion
   <title>Delete unused kernels automatically</title>
```

File: xml/tuning_multikernel.xml [hunk 3]
```diff
@@ -224,7 +224,7 @@
     </step>
    </procedure>
   </sect2>
-  <sect2 xml:id="sec-tuning-multikernel-deleteoldkernel">
+  <sect2 xml:id="sec-admin-multikernel-deleteoldkernel">
    <title>Use case: Deleting an old kernel after reboot only</title>
```
Comment (cwickert) [inline xml/tuning_multikernel.xml:228]:
```suggestion
   <title>Use case: Delete an old kernel after reboot only</title>
```

File: xml/tuning_multikernel.xml [hunk 4]
```diff
@@ -240,7 +240,7 @@
    </para>
   </sect2>
 
-  <sect2 xml:id="sec-tuning-multikernel-fallback">
+  <sect2 xml:id="sec-admin-multikernel-fallback">
    <title>Use case: Keeping older kernels as fallback</title>
```
Comment (cwickert) [inline xml/tuning_multikernel.xml:244]:
```suggestion
   <title>Use case: Keep older kernels as fallback</title>
```

**Review by cwickert — COMMENTED** [review]
Just a test for training

File: xml/tuning_multikernel.xml [hunk 5]
```diff
@@ -338,7 +338,7 @@
    </step>
   </procedure>
  </sect1>
- <sect1 xml:id="sec-tuning-multikernel-zypper">
+ <sect1 xml:id="sec-admin-multikernel-zypper">
   <title>Installing/removing multiple kernel versions with Zypper</title>
```
Comment (cwickert) [inline xml/tuning_multikernel.xml:342]: You forgot to change this, too.
```suggestion
  <title>Install/remove multiple kernel versions with Zypper</title>
```

File: xml/tuning_multikernel.xml [hunk 6]
```diff
@@ -378,7 +378,7 @@ i  | kernel-firmware      | package | 20210208-2.4      | noarch | SLE-Module-Ba
    </step>
   </procedure>
  </sect1>
- <sect1 xml:id="sec-tuning-multikernel-latest" os="osuse">
+ <sect1 xml:id="sec-admin-multikernel-latest" os="osuse">
   <title>
    Installing the Latest Kernel Version from the Repository <literal>Kernel:HEAD</literal>
   </title>
```
Comment (cwickert) [inline xml/tuning_multikernel.xml:384]: Sentence case, no line breaks!
```suggestion
  <title>Installing the Latest Kernel Version from the Repository <literal>Kernel:HEAD</literal></title>
```

**Review by cwickert — COMMENTED** [review]
This time we found an actual bug!

File: xml/tuning_multikernel.xml (hunk 5 above)
Comment (cwickert) [reply xml/tuning_multikernel.xml:342]: No, let's leave it as is!

**Review by dariavladykina — COMMENTED** [review]
Hi Christoph, 

I made some suggestions to the 3 commits of yours in this file, please take a look.

Thanks,
Daria

File: xml/tuning_multikernel.xml [hunk 7]
```diff
@@ -378,10 +378,8 @@ i  | kernel-firmware      | package | 20210208-2.4      | noarch | SLE-Module-Ba
    </step>
   </procedure>
  </sect1>
- <sect1 xml:id="sec-tuning-multikernel-latest" os="osuse">
-  <title>
-   Installing the Latest Kernel Version from the Repository <literal>Kernel:HEAD</literal>
-  </title>
+ <sect1 xml:id="sec-admin-multikernel-latest" os="osuse">
+  <title>Installing the Latest Kernel Version from the Repository <literal>Kernel:HEAD</literal></title>
```
[the remark this answers is not in the document]
Comment (cwickert) [reply xml/tuning_multikernel.xml:382]: I fixed this in 181641d6e90bbaed63108194467d06d621fccb3b

File: xml/tuning_multikernel.xml [hunk 8]
```diff
@@ -50,8 +50,8 @@
    <xref linkend="sec-grub2-yast2-config"/> for more information.
   </para>
  </tip>
- <sect1 xml:id="sec-tuning-multikernel-enable">
-  <title>Enabling and configuring multiversion support</title>
+ <sect1 xml:id="sec-admin-multikernel-enable">
+  <title>Enabe and configure multiversion support</title>
```
[the remark this answers is not in the document]
Comment (cwickert) [reply xml/tuning_multikernel.xml:54]: Good catch!

File: xml/tuning_multikernel.xml [hunk 9]
```diff
@@ -105,8 +105,8 @@
    </para>
   </warning>
 
-  <sect2 xml:id="sec-tuning-multikernel-enable-keep">
-   <title>Automatically deleting unused kernels</title>
+  <sect2 xml:id="sec-admin-multikernel-enable-keep">
+   <title>Delete unused kernels automatically</title>
    <para>
     When frequently testing new kernels with multiversion support enabled, the
```
[the remark this answers is not in the document]
Comment (cwickert) [reply xml/tuning_multikernel.xml:111]: Nope, that would change the meaning. It is kernels, which have multiversion support (enabled), not that multiversion support is getting enabled when testing.

**Comment by cwickert** [comment]
Anyway, enough playing around, I'll close this now.

**Pull request closed without merging**